Scraping bitcoin Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying bitcoin OR BTC since:2018-01-02 until:2018-01-03
Querying bitcoin OR BTC since:2018-01-01 until:2018-01-02
Failed to parse JSON "Expecting value: line 1 column 1 (char 0)" while requesting "https://twitter.com/i/search/timeline?vertical=default&include_available_features=1&include_entities=1&reset_error_state=false&src=typd&max_position=TWEET-948252592349548544-948339903175118849-BD1UO2FFu9QAAAAAAAAVfAAAAAcAAABWAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==-T-0-22&q=bitcoin%20OR%20BTC%20since%3A2018-01-02%20until%3A2018-01-03&l=".
Traceback (most recent call last):
  File "/home/warproxxx/Desktop/Projects/CryptoTrader/src/data_utils/twitter_data/twitterscraper/query.py", line 96, in query_single_page
    json_resp = json.loads(response.text)
  File "/usr/local/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Retrying... (Attempts left: 10)
Failed to parse JSON "Expecting value: line 1 column 1 (char 0)" while requesting "https://twitter.com/i/search/timeline?vertical=default&include_available_features=1&include_entities=1&reset_error_state=false&src=typd&max_position=TWEET-947618371184586752-947975119141507073-BD1UO2FFu9QAAAAAAAAVfAAAAAcAAABWAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==-T-0-49&q=bitcoin%20OR%20BTC%20since%3A2018-01-01%20until%3A2018-01-02&l=".
Traceback (most recent call last):
  File "/home/warproxxx/Desktop/Projects/CryptoTrader/src/data_utils/twitter_data/twitterscraper/query.py", line 96, in query_single_page
    json_resp = json.loads(response.text)
  File "/usr/local/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Retrying... (Attempts left: 10)
Got 995 tweets for bitcoin%20OR%20BTC%20since%3A2018-01-01%20until%3A2018-01-02.
Got 995 tweets (995 new).
ConnectionError ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')) while requesting "https://twitter.com/i/search/timeline?vertical=default&include_available_features=1&include_entities=1&reset_error_state=false&src=typd&max_position=TWEET-948053574021349376-948339903175118849-BD1UO2FFu9QAAAAAAAAVfAAAAAcAAABWAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==-T-0-59&q=bitcoin%20OR%20BTC%20since%3A2018-01-02%20until%3A2018-01-03&l="
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 445, in send
    timeout=timeout
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/lib/python3.7/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/warproxxx/Desktop/Projects/CryptoTrader/src/data_utils/twitter_data/twitterscraper/query.py", line 89, in query_single_page
    response = requests.get(url, headers=headers)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 512, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 622, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 495, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
Retrying... (Attempts left: 10)
Got 1463 tweets for bitcoin%20OR%20BTC%20since%3A2018-01-02%20until%3A2018-01-03.
Got 2458 tweets (1463 new).
Scraping dashcoin Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying dashcoin OR DASH OR darkcoin since:2018-01-02 until:2018-01-03
Querying dashcoin OR DASH OR darkcoin since:2018-01-01 until:2018-01-02
Got 191 tweets for dashcoin%20OR%20DASH%20OR%20darkcoin%20since%3A2018-01-02%20until%3A2018-01-03.
Got 191 tweets (191 new).
Got 292 tweets for dashcoin%20OR%20DASH%20OR%20darkcoin%20since%3A2018-01-01%20until%3A2018-01-02.
Got 483 tweets (292 new).
Scraping dogecoin Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying dogecoin OR DOGE since:2018-01-01 until:2018-01-02
Querying dogecoin OR DOGE since:2018-01-02 until:2018-01-03
Got 75 tweets for dogecoin%20OR%20DOGE%20since%3A2018-01-02%20until%3A2018-01-03.
Got 75 tweets (75 new).
Got 81 tweets for dogecoin%20OR%20DOGE%20since%3A2018-01-01%20until%3A2018-01-02.
Got 156 tweets (81 new).
Scraping ethereum Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying ethereum OR ETH since:2018-01-02 until:2018-01-03
Querying ethereum OR ETH since:2018-01-01 until:2018-01-02
Got 242 tweets for ethereum%20OR%20ETH%20since%3A2018-01-01%20until%3A2018-01-02.
Got 242 tweets (242 new).
Got 413 tweets for ethereum%20OR%20ETH%20since%3A2018-01-02%20until%3A2018-01-03.
Got 655 tweets (413 new).
Scraping litecoin Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying litecoin OR LTC since:2018-01-01 until:2018-01-02
Querying litecoin OR LTC since:2018-01-02 until:2018-01-03
Got 142 tweets for litecoin%20OR%20LTC%20since%3A2018-01-01%20until%3A2018-01-02.
Got 142 tweets (142 new).
Got 214 tweets for litecoin%20OR%20LTC%20since%3A2018-01-02%20until%3A2018-01-03.
Got 356 tweets (214 new).
Scraping monero Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying monero OR XMR since:2018-01-02 until:2018-01-03
Querying monero OR XMR since:2018-01-01 until:2018-01-02
Got 79 tweets for monero%20OR%20XMR%20since%3A2018-01-01%20until%3A2018-01-02.
Got 79 tweets (79 new).
Got 89 tweets for monero%20OR%20XMR%20since%3A2018-01-02%20until%3A2018-01-03.
Got 168 tweets (89 new).
Scraping ripple Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying ripple OR XRP since:2018-01-01 until:2018-01-02
Querying ripple OR XRP since:2018-01-02 until:2018-01-03
Got 267 tweets for ripple%20OR%20XRP%20since%3A2018-01-01%20until%3A2018-01-02.
Got 267 tweets (267 new).
Got 409 tweets for ripple%20OR%20XRP%20since%3A2018-01-02%20until%3A2018-01-03.
Got 676 tweets (409 new).
Scraping stellar Data
Starting Year: 2018 Ending Year: 2018
Current Starting Date:2018-01-01 Current Ending Date:2018-01-03
2
Querying stellar OR STR since:2018-01-01 until:2018-01-02
Querying stellar OR STR since:2018-01-02 until:2018-01-03
Got 157 tweets for stellar%20OR%20STR%20since%3A2018-01-01%20until%3A2018-01-02.
Got 157 tweets (157 new).
Got 141 tweets for stellar%20OR%20STR%20since%3A2018-01-02%20until%3A2018-01-03.
Got 298 tweets (141 new).
